src/lottie/lottieparser.cpp

```diff
@@ -987,6 +987,10 @@ std::shared_ptr<LOTMaskData> LottieParserImpl::parseMaskObject()
             obj->mInv = GetBool();
         } else if (0 == strcmp(key, "mode")) {
             const char *str = GetString();
+            if (!str) {
+                Skip(key);
```

Comment:
Skip() will skip the next (key,value) which is not desirable
the proper fix could be 
if (!str) {
    ob->mMode = LOTMaskData::Mode::None;
    continue;
}

Could you please update the patch. thanks